src/registry_schemas/schemas/party.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/parties",
+    "type": "object",
+    "title": "The Corp Parties Schema",
+    "definitions": {
+        "person": {
+            "type": "object",
+            "properties": {
+                "partyType": {
+                    "type": "string",
+                    "title": "The type of party",
+                    "enum": [
+                        "org",
+                        "person"
+                    ]
+                },
+                "firstName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "lastName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "middleInitial": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "orgName": {
+                    "type": "string",
+                    "maxLength": 50
+                },
+                "orgType": {
```

Comment:
I wasn't aware of this property. What is this used for? Is this something we should be putting into the filing?

Reply:
This is to differentiate between a person and an organisation

Reply:
Isn't that what `partyType` is?

Reply:
should be resolved